PeerioTechnologies/peerio-desktop
# chore: replace feedback chat with support

#### Relevant info and issue/PR links



#### Testing instructions
click chat button next to Chat with us! in feedback section
should open chat with @support

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
